packages/shylu/shylu_dd/frosch/src/SchwarzOperators/FROSch_HarmonicCoarseOperator_def.hpp

```diff
@@ -283,27 +295,85 @@ namespace FROSch {
         ExtensionSolver_->initialize();
         ExtensionSolver_->compute();
         ExtensionSolver_->apply(*mVtmp,*mVPhiI);
+
+        GOVec priorIndex(NumberOfBlocks_,0);
+        GOVec postIndex(NumberOfBlocks_,0);
+        
+        GOVec2D excludeCols(NumberOfBlocks_);
```

Comment:
Do we currently need `excludeCols` or is it, again, related to, e.g., Lagrange multipliers?

Reply:
It's for off-diagonal blocks, which can be exluded.